server/processors/src/main/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriver.java

```diff
@@ -116,7 +123,8 @@ protected void deriveNodeDetails(Trace trace, List<Node> nodes, List<NodeDetails
                     nd.setHostName(trace.getHostName());
                 }
 
-                nd.setProperties(trace.allProperties());
```

Comment:
Added an appropriate comment - let me know if ok for now, before HAPM-778 resolved.